### PR TITLE
Update JuniDB.md

### PR DESCRIPTION
**Pull request by andskur**
https://github.com/w3f/Grants-Program/pull/466

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).

### CODE REVIEW

**Comment by mmagician** [comment]
@andskur You removed the key deliverable of Milestone 1. It looks like your milestone now only includes *forking* substrate, I don't see how we would pay $7,000 for this.

**Comment by alxs** [comment]
@mmagician I think the purpose is to update [offchain::ipfs](https://github.com/rs-ipfs/substrate) to Substrate v3, or at least that's what I read from [here](https://github.com/w3f/Grant-Milestone-Delivery/pull/254/files#diff-9fee4ec1fe0f082883f36b07b01dd6fe9b0d8804be9c694a35ce5c5820a53f4cR20). But @andskur I would first clarify that the team behind offchain::ips is ok with that (they may want to do this themselves soon) and I'd argue that your work should be merged into offchain::ipfs as part of the milestone instead of it just living in a separate fork.

**Review by mmagician — DISMISSED** [review]
My bad @andskur @MikeMS-sys, I have somehow missed the connection between the delivery and this amendment. Makes absolute sense and as I had written before, I'm totally supportive of rejuvenating the offchain::ipfs repo.

**Review by Noc2 — DISMISSED** [review]
Makes sense and https://equilibrium.co/ so far didn’t send us anything. So, I think they are busy doing other things.

**Comment by Noc2** [comment]
@andskur Btw. it looks like you still need to sign the terms and conditions.

File: applications/JuniDB.md [hunk 1]
```diff
@@ -222,9 +222,8 @@ Link to 2nd pull request
 | 0b. | Documentation | Documentation of the code and a basic tutorial describing how the software can be used and tested. |
 | 0c. | Testing Guide | Complex quality assurance for all platform features. Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
 | 0d. | Article | We will publish an article that explains the functionality of the proposed pallet in this milestone. |
-| 1. | Substrate pallet | Add hash data storage to the pallet. Update rpc module to interact with new data storage. Database optimizations. |  
-| 2. | Web application | Update web application with new data storage rpc. |  
-| 3. | Docker image| We will provide a dockerfile to demonstrate the full functionality of testing Substrate chain with integrated Database pallet. |
+| 1. | Substrate fork | Substrate last version fork with fully updated version of offchain::ipfs. |
```
Comment (alxs) [inline applications/JuniDB.md:225]: Sounds good. Could you mention this explicitly? 

```suggestion
| 1. | Update [offchain::ipfs](https://github.com/rs-ipfs/substrate) | We will coordinate with equilibrium.co to update offchain::ipfs to the latest Substrate version via PR to the existing repository |
```

**Review by alxs — CHANGES_REQUESTED** [review]

**Comment by alxs** [comment]
And could you please update the total cost and duration too?

**Review by alxs — APPROVED** [review]

**Comment by MikeMS-sys** [comment]
Hi all,

@mmagician  It's ok )
@Noc2 Fixed it.
@alxs Made changes.

**Review by mmagician — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Pull request merged**